projectroles/migrations/0026_delete_category_settings.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.2.18 on 2023-03-14 10:21
+
+from django.db import migrations
+from projectroles.app_settings import AppSettingAPI
+
+
+def clean_up_app_settings(apps, schema_editor):
+    app_settings_api = AppSettingAPI()
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+
+    # Find all app settings with project type 'CATEGORY'
+    app_settings = AppSetting.objects.exclude(project=None)
+
+    for app_setting in app_settings:
+        try:
+            setting_def = app_settings_api.get_definition(
+                app_name=app_setting.app_plugin.name, name=app_setting.name
+            )
+        except ValueError:
+            continue
```

Comment:
We should also delete the AppSetting if no definition is found. Also, we can simplify this by doing just one `get_definition()` call and first set `app_name` as follows:

`app_name = app_setting.app_plugin.name if app_setting.app_plugin else 'projectroles'`